cms/release_calendar/forms.py

```diff
@@ -13,6 +14,9 @@
 from .enums import LOCKED_STATUS_STATUSES, NON_PROVISIONAL_STATUS_CHOICES, ReleaseStatus
 from .utils import get_translated_string, parse_day_month_year_time, parse_month_year
 
+if TYPE_CHECKING:
+    from .models import ReleaseCalendarPage
+
 DATE_SEPERATOR = {
```

Comment:
not in scope of this PR, but would be nice to fix: `DATE_SEPERATOR` -> `DATE_SEPARATOR`

Reply:
We definitely have to fix this part of this PR. Unacceptable 🫣